File: c/learnc/ex13.c
#include <stdio.h>

int main(int argc, char *argv[])
{
	int i = 0;

	// go through each string in argv
	// why am I skipping argv[0]?
	// because that is the name of the file
	for (i = 1; i < argc; i++) {
		printf("arg %d: %s\n", i, argv[i]);
	}

	//let's make our own array of strings
	char *states[] = {
		"California", "Oregon",
		"Washington", "Virginia",
		"Maryland", "Delaware"
	};

	int num_states = 6;

	for (i = 0; i < num_states; i++) {
		printf("state %d: %s\n", i, states[i]);
	}

	return 0;
}

File: c/learnc/ex13.js
function main() {
	var i = 0;

	for (i = 2; i < process.argv.length; i++) {
		console.log(`arg ${i} :  ${process.argv[i]}`);
	}

	var states = ["California", "Oregon", "Washington", "Texas"];

	var num_states = 4;

	for (i = 0; i < num_states; i++) {
		console.log("state " + i + ": " + states[i]);
	}

	return false;
}

main();
